.circleci/config.yml

```diff
@@ -121,6 +121,88 @@ jobs:
     docker:
       - image: cimg/python:3.6
 
+  test-install-pip-python-36: &test-install-pip
+    docker:
+      - image: cimg/python:3.6
+
+    steps:
+
+      - run:
+          name: install-with-pip
+          command: |
+            pip install --progress-bar off signac signac-flow
+
+      - run: &test-run-and-import
+          name: test-run-and-import
+          command: |
+            signac --version
+            python -c 'import signac'
+            flow --version
+            python -c 'import flow'
```

Comment:
Do we know if importing is enough to catch the issues we want to catch? This would not catch issues that rely on actually executing code.

Reply:
It's a smoke test. We can expand that to more tests if we like, but I would prefer to leave it as is.